src/pyspiffe/svid/jwt_svid.py

```diff
@@ -115,19 +115,26 @@ def parse_and_validate(
             validator = JwtSvidValidator()
             validator.validate_header(token_header)
             signing_key = jwt_bundle.find_jwt_authority(token_header['kid'])
+
+            public_key_pem = signing_key.public_bytes(
+                encoding=serialization.Encoding.PEM,
+                format=serialization.PublicFormat.SubjectPublicKeyInfo,
+            ).decode('UTF-8')
+
             claims = jwt.decode(
                 token,
-                algorithms=token_header['alg'],
-                key=signing_key,
+                algorithms=token_header.get('alg', None),
+                key=public_key_pem,
                 audience=audience,
                 options={
                     'verify_signature': True,
                     'verify_aud': True,
                     'verify_exp': True,
                 },
             )
-            # TODO:validate required claims
             spiffe_ID = SpiffeId.parse(claims['sub'])
             return JwtSvid(spiffe_ID, claims['aud'], claims['exp'], claims, token)
```

Comment:
I know this is not part of your changes but I think it worth the question.
Is it possible that the token do not contains those claims? In that case we should use get() to avoid a `KeyError`

Reply:
That is a good catch, Andres! Thanks!

Reply:
The object `options` passed as argument to `jwt.decode()` will enforce the presence of those claims. 
Thanks Glaucimar for the explanation.